queries/runescript/highlights.scm

```diff
@@ -0,0 +1,92 @@
+; Variables
+[
+  (identifier)
+  (local_variable)
+  (game_variable)
+  (constant_variable)
+  (parameter_variable)
+  (declaration_variable)
+] @variable
+
+; Scripts
+(script
+  trigger: (identifier) @keyword.directive
+  subject: (identifier) @function)
+
+; Calls
+(call
+  [
+    "~"
+    "@"
+  ]? @function
+  callee: (identifier) @function)
+
+; Literals
+((literal) @constant.builtin
+  (#any-of? @constant.builtin "true" "false" "null"))
+
+[
+  (integer_literal)
+  (coord_literal)
+] @number
+
+(comment) @comment
+
+(string_fragment) @string
+
+(string_interpolation
+  [
+    "<"
+    ">"
+  ] @punctuation.special)
+
+(string_tag) @punctuation.special
+
+; Types
+(type) @type
+
+(parameter_type) @type
+
+; Tokens
+[
+  ";"
+  ":"
+  ","
+] @punctuation.delimiter
+
+[
+  "("
+  ")"
+  "{"
+  "}"
+] @punctuation.bracket
+
+[
+  "*"
+  "/"
+  "+"
+  "-"
+  "&"
+  "|"
+  ">"
+  ">="
+  "<"
+  "<="
+  "="
+  "!"
+] @operator
+
+(arithmetic
+  "%" @operator)
+
+[
+  "default"
+  "if"
+  "else"
```

Comment:
these should be `@keyword.conditional` (and similarly for the others; please use as specific a capture -- from the list in CONTRIBUTING.md -- as possible).